test_collections/matter/scripts/OTBR/otbr_start.sh

```diff
@@ -63,15 +70,17 @@ BR_PARAMS=(
 "dataset active -x"
 )
 
+print_script_step "Setting up Thread Network"
+
 for i in "${BR_PARAMS[@]}"
 do
-        echo "Param: '"$i"'"
+        printf "Param: '$i'"
         sudo docker exec -t otbr-chip ot-ctl $i || exit 1
 done
 
 BR_SIMPLE_DATASET="00030000"${BR_CHANNEL_HEX}"0208"${BR_EXTPANID}"0510"${BR_NETWORKKEY}"0102"${BR_PANID}
 sudo echo ${BR_SIMPLE_DATASET} > /tmp/otbr_simple_dataset.txt
-echo "Simple Dataset: " ${BR_SIMPLE_DATASET}
+printf "Simple Dataset: $BR_SIMPLE_DATASET"
 
-# Also make sure to restart the Raspi avahi to have it in a clean state
+print_script_step "Also make sure to restart the Raspi avahi to have it in a clean state"
 sudo service avahi-daemon restart
```

Comment:
Should there be a `print_end_of_script` at the end here?

Reply:
In this case no...  because we want so send this message to the user.

But we need to add the print_end_of_script  in the end.